great_expectations/datasource/fluent/sources.py

```diff
@@ -271,8 +271,27 @@ def _bind_asset_factory_method_if_not_present(
             def _add_asset_factory(
                 self: Datasource, name: str, **kwargs
             ) -> pydantic.BaseModel:
+
+                # if the data_connector uses a data_connector we need to identify the
```

Comment:
@Kilo59 This sentence reads "if the data_connector uses a data_connector" -- does it need re-phrasing?  If you are documenting the `connect_options` here, that is fine; I just wanted to make sure I understood it.  Thanks.

Reply:
Yes thank you. It should be `# if the Datasource uses a data_connector we need to identify the...`